components/browser/engine-system/src/test/java/mozilla/components/browser/engine/system/SystemEngineViewTest.kt

```diff
@@ -73,10 +73,10 @@ class SystemEngineViewTest {
 
         engineView.currentWebView.webViewClient.onPageStarted(null, "http://mozilla.org", null)
         assertEquals(true, observedLoadingState)
+        assertEquals("http://mozilla.org", observedUrl)
 
         observedLoadingState = true
         engineView.currentWebView.webViewClient.onPageFinished(null, "http://mozilla.org")
-        assertEquals("http://mozilla.org", observedUrl)
```

Comment:
Can we use two different URLs here to make sure both changes are observed?

Reply:
👍